web/src/main/java/it/pagopa/selfcare/dashboard/web/model/user/UserProductRoles.java

```diff
@@ -2,11 +2,9 @@
 
 import lombok.Data;
 
-import javax.validation.constraints.NotEmpty;
 import java.util.Set;
 
 @Data
 public class UserProductRoles {
-    @NotEmpty
     Set<String> productRoles;
```

Comment:
this object is used for output, we shouldn't take away the validation...

https://github.com/pagopa/selfcare-dashboard-backend/blob/48e0f966820936838e64cd03b78224cef2a5edb7/web/src/main/java/it/pagopa/selfcare/dashboard/web/controller/InstitutionController.java